ui/src/timeMachine/components/variableToolbar/VariableTooltipContents.tsx

```diff
@@ -67,56 +61,56 @@ const VariableTooltipContents: FunctionComponent<Props> = ({
     onAddVariableToTimeMachine(variableID)
   }
 
-  let selectedOption = 'None Selected'
-  let icon
   let status = toComponentStatus(valuesStatus)
-  // this should set the selectedKey as the key
-  // set as a ternary in order to get e2e test to pass since values are undefined on the test
-  const key = values && values.selectedKey ? values.selectedKey : undefined
 
-  if (!values) {
-    selectedOption = 'Failed to Load'
-    icon = IconFont.AlertTriangle
-    status = ComponentStatus.Disabled
-  } else if (values.error) {
-    selectedOption = 'Failed to Load'
-    icon = IconFont.AlertTriangle
+  if (selectedValue === 'Failed to Load') {
     status = ComponentStatus.Disabled
-  } else if (key === undefined || values.values[key] === undefined) {
-    selectedOption = 'No Results'
-  } else {
-    selectedOption = get(values, 'selectedKey', 'None Selected')
   }
 
-  const button = () => (
-    <Dropdown.Button status={ComponentStatus.Disabled} onClick={() => {}}>
-      {selectedOption}
-    </Dropdown.Button>
-  )
-
-  const menu = () => null
+  const handleSelect = (selectedValue: string) => {
+    onSelectVariableValue(variableID, selectedValue)
+  }
 
   return (
     <div onMouseEnter={handleMouseEnter}>
       <Form.Element label="Value">
-        {dropdownItems.length === 1 ? (
-          <Dropdown
-            button={button}
-            style={{width: '200px'}}
-            menu={menu}
-            testID="variable--tooltip-dropdown"
-          />
-        ) : (
-          <SelectDropdown
-            buttonIcon={icon}
-            options={dropdownItems as string[]}
-            selectedOption={selectedOption}
-            testID="variable--tooltip-dropdown"
-            buttonStatus={status}
-            style={{width: '200px'}}
-            onSelect={value => onSelectVariableValue(variableID, value)}
-          />
-        )}
+        <Dropdown
+          style={{width: '200px'}}
+          testID="variable--tooltip-dropdown"
+          button={(active, onClick) => (
+            <Dropdown.Button
+              active={active}
+              onClick={onClick}
+              testID="variable-dropdown--button"
+              status={status}
```

Comment:
got user feedback that it would be helpful to disable the dropdown if there's only 1 option available. Not gonna push for it this time but something to keep in mind moving forward